apps/ledger-live-desktop/src/internal/commands/publicKeyTweakAdd.js

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
why using flow here don't seem used. Also no way to use ts here ?

Reply:
I can probably drop this comment but this will still be a .js as it's the current status quo on LLD and I have only copying another existing commands/* , this also is temporary workaround until we make a webworker for this. (migrating to TS is a wider scope)

Reply:
ok i just saw that SOME commands are actually .ts file, i'll see what i can do